### DISCUSSION TITLE
Github actions stuck?

### DISCUSSION THREAD

**Discussion by ryanciprianiTR** [community/community#146172] [Actions]
### Select Topic Area

Are github.com actions stuck? it's not sending to our runners that were working fine earlier today.

### Body

Hi all. We use self hosted runners for a project.
About 42 minutes ago, github stopped sending jobs to the runner.

Last entry is:
"Waiting for a runner to pick up this job..."

The runner looks fine, the jobs ar ethe same as those that previously ran, it just looks like github stopped sending to the runner.

`[thisuser@ip-XXIPXX XXMYREPOXX-ip-XXIPXX]$ ./config.sh --check
What is the URL of your repository? https://github.com/MYCMP/XXMYREPOXX/
What is your GitHub personal access token? ****************************************
**********************************************************************************************************************
**  Check:               Internet Connection
**  Description:         Check if the Actions runner has internet access.
**********************************************************************************************************************
**                                                                                                                  **
**                                            P A S S                                                               **
**                                                                                                                  **
**********************************************************************************************************************
** Log: /home/github-runner/XXMYREPOXX-ip-XXIPXX/_diag/InternetCheck_20241203-190133-utc.log
**********************************************************************************************************************

**********************************************************************************************************************
**  Check:               GitHub Actions Connection
**  Description:         Check if the Actions runner has access to the GitHub Actions service.
**********************************************************************************************************************
**                                                                                                                  **
**                                            P A S S                                                               **
**                                                                                                                  **
**********************************************************************************************************************
** Log: /home/github-runner/XXMYREPOXX-ip-XXIPXX/_diag/ActionsCheck_20241203-190133-utc.log
**********************************************************************************************************************

**********************************************************************************************************************
**  Check:               Git Certificate/Proxy Validation
**  Description:         Check if the Git CLI can access GitHub.com or GitHub Enterprise Server.
**********************************************************************************************************************
**                                                                                                                  **
**                                            P A S S                                                               **
**                                                                                                                  **
**********************************************************************************************************************
** Log: /home/github-runner/XXMYREPOXX-ip-XXIPXX/_diag/GitCheck_20241203-190133-utc.log
**********************************************************************************************************************

**********************************************************************************************************************
**  Check:               Node.js Certificate/Proxy Validation
**  Description:         Check if Node.js has access to GitHub.com or GitHub Enterprise Server.
**********************************************************************************************************************
**                                                                                                                  **
**                                            P A S S                                                               **
**                                                                                                                  **
**********************************************************************************************************************
** Log: /home/github-runner/XXMYREPOXX-ip-XXIPXX/_diag/NodeJsCheck_20241203-190133-utc.log
**********************************************************************************************************************
`

**Comment by ryanciprianiTR** [comment]
Looks like the issue was the runner

stopped and started the runner and it started picking up again - just nothing in the logs to indicate what the issue was.

**Comment by feccruz** [comment]
Somos todos apenas uma unidade divina.

Yahoo Mail: Busque, organize e aumente sua produtividade 
 
  Em ter., 3 3e dez. 3e 2024 às 15:20, ***@***.***> escreveu:   


Closed #146172 as resolved.
